launcher/ui/pages/global/APIPage.ui

```diff
@@ -220,6 +220,31 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="groupBox_ua">
+         <property name="minimumSize">
+          <size>
+           <width>0</width>
+           <height>0</height>
+          </size>
+         </property>
+         <property name="title">
+          <string>User Agent</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_7">
+          <item>
+           <widget class="QLineEdit" name="userAgentLineEdit"/>
+          </item>
+          <item>
+           <widget class="QLabel" name="userAgentLabel">
+            <property name="text">
+             <string>Enter a custom User Agent here. The special string ${launcher_version} will be replaced with the version of the launcher.</string>
```

Comment:
maybe `$LAUNCHER_VER` would be better, to keep things similar to the ones in the custom commands page

Reply:
hm okay

Reply:
(anyways, this isn't even actually implemented)

Reply:
now it is